# the current status for a channel on the motor controller

uint8 channel
int32 position
float64 velocity
float64 current
sensor_msgs/Temperature temp